<?xml version="1.0"?>
<!--
Demo launch file to run single softkinetic camera in stand alone (no other /tf) setup with Rviz
The arguments given are the device indices of the cameras determined by the DepthSense deamon.
-->
<launch>
<!--  <param name="m_parent_frame" type="string" value="/camera_link"/> -->

  <node pkg="door_handle_detection" type="door_handle_detection_node" name="door_handle_detection" output="screen">
    <param name="dh_right" type="bool" value="true"/>
    <param name="parent_tf" type="string" value="camera_depth_optical_frame"/>
    <param name="pclTopicName" type="string" value="/camera/depth/points"/>
    <param name="cameraRGBTopicName" type="string" value="/camera/rgb/camera_info"/>
    <param name="cameraDepthTopicName" type="string" value="/camera/depth/camera_info"/>
    <param name="imageTopicName" type="string" value="/camera/rgb/image_raw"/>
    <param name="debug" type="bool" value="true"/>
  </node>

  
</launch>
